listenbrainz/webserver/views/api.py

```diff
@@ -375,13 +375,19 @@ def latest_import():
 
 
 @api_bp.route('/validate-token', methods=['GET'])
+@crossdomain(headers='Authorization')
 @ratelimit()
 def validate_token():
     """
     Check whether a User Token is a valid entry in the database.
 
-    In order to query this endpoint, send a GET request with the token to check
-    as the `token` argument (example: /validate-token?token=token-to-check)
+    In order to query this endpoint, send a GET request with the token in
+    the Authorization header.
```

Comment:
Can we open a ticket to improve the documentation for authentication? We currently have this text in most endpoints:
>   In order to update the timestamp of a user, you'll have to provide a user token in the Authorization
>   Header. User tokens can be found on https://listenbrainz.org/profile/ .

It'd be nice to have a separate "authorization" section in the docs and link all endpoints to this.

Reply:
Done. LB-861